cfme/tests/infrastructure/test_infra_manual.py

```diff
@@ -288,39 +288,232 @@ def test_host_comparison_my_company_tags():
+            3. The "Compare Host / Node" view should be displayed.
+               - icons are displayed for all and only selected hosts with hostname displayed
+               - one of the quadicons has the host denoted as "(base)" in the hostname
+               - host properties row is displayed (default)
+               - "% Matched" text or graphs are displayed
+               - remove icons exist for all non-base hosts.
+            4. The clicked host should be removed from display.
+               - The same host is still denoted as the base host.
+               - when, and only when, 3 or more hosts are displayed, remove icons exist for all
+                 non-base hosts.
+            5. The clicked host should be removed from display.
+               - The same host is still denoted as the base host.
+               - when, and only when, 3 or more hosts are displayed, remove icons exist for all
+                 non-base hosts.
+            6. Hosts should be displayed without the remove host icon.
+    """
+    pass
+
+
+@test_requirements.infra_hosts
+def test_host_comparison_exists_mode():
+    """
+    Polarion:
+        assignee: prichard
+        casecomponent: Infra
+        caseimportance: low
+        initialEstimate: 1/4h
+        testSteps:
+            1. Navigate to the Compute > Infrastructure > Hosts view.
+            2. Select at least 3 hosts (more if possible) by checking the box in upper left of
+               quadicons.
+            3. Click "Compare Selected Items" under the "Configuration" dropdown.
+            4. Select "Network Adapters" from the compare section accordion and click "Apply".
+            5. Click on the "Network Adapters" row, in the Compare Host/Node view, to expand it.
+            6. Click the "Exists Mode" icon (in row of buttons to top left).
+        expectedResults:
+            1. Hosts view is displayed.
+            2. The selected hosts should be displayed with a blue border and checked checkbox.
+            3. The "Compare Host / Node" view should be displayed (in expanded view by default).
+               - icons are displayed for all and only selected hosts with hostname displayed
+               - one of the quadicons has the host denoted as "(base)" in the hostname
+               - host properties row is displayed (default)
+               - "% Matched" text or graphs are displayed
+               - remove icons exist for all non-base hosts.
+            4. Network Adapter row will be displayed in the Compare Host/Node panel.
+            5. Individual child rows will be displayed with "% Matched" in base host and graphs
+               displayed for the rest of the hosts.
+            6. The "% Matched" and graphs displayed on child rows will be replaced with "+" or
+                  "-" depending if the item exists in the non-base hosts.
+    """
+    pass
+
+
+@test_requirements.infra_hosts
+def test_host_comparison_compressed_view():
+    """
+        Polarion:
```

Comment:
Required: this is indented one block too far

Reply:
updated as suggested